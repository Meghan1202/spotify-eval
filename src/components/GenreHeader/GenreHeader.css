.Genre-Header-Container {
    position: relative;
    height: 7rem;
    width: 9rem;
    display: flex;
    margin: 0.625rem;
}

.Genre-Image {
    height: 7rem;
    width: 7rem;
    border-radius: 7rem;
}

.Genre-Indicator {
    background-color: #4C63AA;
    color: white;
    border-radius: 1rem;
    width: fit-content;
    height: fit-content;
    padding: 0.62rem;
    position: absolute;
    bottom: 35%;
    right: 10%;
}