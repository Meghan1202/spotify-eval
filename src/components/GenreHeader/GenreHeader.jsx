import React from 'react';
import './GenreHeader.css';

const GenreHeader = () => (
  <div className="Genre-Header-Container">
    <img className="Genre-Image" src="./assets/genre-bollywood.png" alt="genre" />
    <div className="Genre-Indicator">Bollywood</div>
  </div>
);

export default GenreHeader;
