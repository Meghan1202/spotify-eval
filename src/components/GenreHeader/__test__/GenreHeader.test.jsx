import React from 'react';
import { render } from '@testing-library/react';
import GenreHeader from '../GenreHeader';

describe(GenreHeader.name, () => {
  it('should match the snapshot', () => {
    const { container } = render(<GenreHeader Genre="Bollywood" />);
    expect(container).toMatchSnapshot();
  });
});
