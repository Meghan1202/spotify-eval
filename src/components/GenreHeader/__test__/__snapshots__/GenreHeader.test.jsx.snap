// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreHeader should match the snapshot 1`] = `
<div>
  <div
    class="Genre-Header-Container"
  >
    <img
      alt="genre"
      class="Genre-Image"
      src="./assets/genre-Bollywood.png"
    />
    <div
      class="Genre-Indicator"
    >
      Bollywood
    </div>
  </div>
</div>
`;
